webapp/src/AppComponents/DeleteWorkbookDialog.tsx

```diff
@@ -139,22 +162,24 @@ const Button = styled.button`
   text-overflow: ellipsis;
   transition: background-color 150ms;
   &:hover {
-    background-color: ${theme.palette.primary.dark};
+    background-color: ${PRIMARY_DARK};
   }
 `;
 
 const DeleteButton = styled(Button)`
-  background-color: ${theme.palette.error.main};
-  color: ${theme.palette.common.white};
+  background-color: ${ERROR_MAIN};
+  color: ${COMMON_WHITE};
   &:hover {
-    background-color: ${theme.palette.error.dark};
+    background-color: ${ERROR_DARK};
   }
 `;
 
 const CancelButton = styled(Button)`
-  background-color: ${theme.palette.grey["200"]};
-  color: ${theme.palette.grey["700"]};
+  background-color: ${GREY_200};
+  color: ${GREY_700};
   &:hover {
-    background-color: ${theme.palette.grey["300"]};
+    background-color: ${GREY_300};
   }
 `;
+
+export default DeleteWorkbookDialog;
```

Comment:
For things like that, there is only one export, we want to use the default export (so we use `import myDialog from ..` instead of `import { myDialog } from ..`